docs/DevelopmentSetup/visual_studio_win_example.rst

```diff
@@ -74,219 +78,267 @@ Commands to run cmake for the spatial simulations:
     # Generate the project files
     cmake -G "Visual Studio 16 2019" -DCMAKE_INSTALL_PREFIX=C:\Development\Software\moja -DVCPKG_TARGET_TRIPLET=x64-windows -DOPENSSL_ROOT_DIR=c:\Development\moja-global\vcpkg\installed\x64-windows -DENABLE_TESTS=OFF -DENABLE_MOJA.MODULES.GDAL=ON -DCMAKE_TOOLCHAIN_FILE=c:\Development\moja-global\vcpkg\scripts\buildsystems\vcpkg.cmake ..
 
+.. _Cmake: ../prerequisites/cmake.html
+.. _Visual Studio: ../prerequisites/visual_studio.html
+.. _Docker: ../prerequisites/docker.html
+.. _FLINT core repository: https://github.com/moja-global/FLINT
+.. _FLINT.example repository: https://github.com/moja-global/FLINT.Example
 
 Running the project
--------------------
+===================
 
 In order to run and debug the Visual Studio solution -
 
-* Open the visual studio solution that CMake created at ``C:\Development\moja-global\FLINT\Source\build\moja.sln``
-
-* Build the debug configuration ``ALL_BUILD`` target by right clicking the ``ALL_BUILD`` node and selecting ``Build``.
+-  Open the visual studio solution that CMake created at
+   ``C:\Development\moja-global\FLINT\Source\build\moja.sln``
+-  Build the debug configuration ``ALL_BUILD`` target by right clicking
+   the ``ALL_BUILD`` node and selecting ``Build``.
 
 .. figure:: ../images/installation_vs2019_flint.example/VS2019_buildall.jpeg
-  :width: 600
-  :align: center
-  :alt: Navigating to Visual Studio Debugging ``ALL_BUILD`` properties page
+   :alt: Navigating to Visual Studio Debugging ``ALL_BUILD`` properties
+   page
+   :align: center
+   :width: 600px
 
-  Navigating to Visual Studio Debugging ``ALL_BUILD`` properties page
+   Navigating to Visual Studio Debugging ``ALL_BUILD`` properties page
 
 .. figure:: ../images/installation_vs2019_flint.example/Step4.png
-  :width: 600
-  :align: center
```

Comment:
Screenshot not visible!
https://moja-global-documentation--73.org.readthedocs.build/en/73/DevelopmentSetup/visual_studio_win_example.html